pkg/utils/urls.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package utils
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+)
+
+// ValidateURL takes in a url and returns a boolean if it's valid
+func ValidateURL(dst string) bool {
+	parsed, err := url.Parse(dst)
```

Comment:
our paths are not URL-s, see examples in `fpath`

Reply:
This was an assumption on my part, I'll update the tests and the code to reflect this.